python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -99,20 +131,26 @@ def fix_date_elements_folder_path(partitions, types):
     return fixed_partitions
 
 
-def complete_file_path_time_pattern(dku_flow_variables, file_path_pattern):
+def complete_file_path_time_pattern(dku_flow_variables, file_path_pattern, source=None):
     """Fill the placeholders of the partition path pattern for the time dimensions with the right partition values.
 
     Args:
         dku_flow_variables (dict): Dictionary of flow variables for a project.
         file_path_pattern (str)
+        source (str): folder id if the folder is a source, None if for destination folder
 
     Returns:
         File path prefix.
     """
     file_path = file_path_pattern
     for time_dimension in TIME_DIMENSION_PATTERNS:
-        time_value = dku_flow_variables.get(time_dimension)
+        time_value = get_dimension_value_from_flow_variables(dku_flow_variables, source, time_dimension)
         if time_value is not None:
             time_pattern = TIME_DIMENSION_PATTERNS.get(time_dimension)
             file_path = file_path.replace(time_pattern, time_value)
     return file_path
+
+
+def get_dimension_value_from_flow_variables(dku_flow_variables, source, dimension):
+    origin = "DST" if source is None else "SRC_{}".format(source)
```

Comment:
Same comment as previously, I would advise using the terms `input/output` instead of `origin/source/destination` which are not used in DSS.